Sources/Rendering/OpenGL/Renderer/index.js

```diff
@@ -27,6 +27,7 @@ function vtkOpenGLRenderer(publicAPI, model) {
       publicAPI.updateLights();
       publicAPI.prepareNodes();
       publicAPI.addMissingNode(model.renderable.getActiveCamera());
+      publicAPI.addMissingPropNodes(model.renderable.getViewProps());
```

Comment:
@martinken does it make sense to have that new `addMissingPropNodes`?

Reply:
Let me look at it when I get a sec. Sorry I missed this.

Reply:
@martinken any thoughts?

Reply:
You should yank the next few lines as they are useless and just consume time. Actors/Volumes etc are all viewprops so you are trying to add them twice from what I can see.